test/utils/scheduler/event-builder.ts

```diff
@@ -146,8 +148,8 @@ export class EventBuilder {
   /**
    * Create a single-day timed event starting at `start` with the given duration (minutes).
    */
-  singleDay(start: string, durationMinutes = 60) {
-    const startDate = this.adapter.date(start, 'default');
+  singleDay(start: string | SchedulerValidDate, durationMinutes = 60) {
```

Comment:
It made my life easier on some tests where the start is derived from another date.